torchaudio/datasets/speechcommands.py

```diff
@@ -36,9 +36,16 @@ def load_speechcommands_item(filepath: str, path: str) -> Tuple[Tensor, int, str
 
 
 class SPEECHCOMMANDS(Dataset):
-    """
-    Create a Dataset for Speech Commands. Each item is a tuple of the form:
-    waveform, sample_rate, label, speaker_id, utterance_number
+    """Create a Dataset for Speech Commands.
+
+    Args:
+        root (str): Path to the directory where the dataset is found or downloaded.
+        url (str, optional): Type of the dataset to dowload. This is **NOT** the actual URL.
+            (default: ``"speech_commands_v0.02"``)
```

Comment:
This could also use a more complete enumeration of the options (and possibly what each means). That can be a separate PR, too.

Reply:
Updated as part of this PR.